dante.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.20
```

Comment:
What is special about our image? Ideally if we coule use a stock one we wouldn't need a github workflow etc...

Reply:
The one i used before was also very old and the creator was using « ash » (making it different from our others images, breaking k8s_ssh for example)

I’m not a fan of the extra workflow either but we shouldn’t have to run it often.

Reply:
Ok if there's no official image then LGTM 👍